src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/osgi/OSGiConfiguration.kt

```diff
@@ -0,0 +1,38 @@
+package com.cognifide.gradle.aem.common.instance.service.osgi
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class OSGiConfiguration {
```

Comment:
only first letter uppercase / make it capitalized;

how does `OsgiConfiguration` and `ConfigurationState` correlate ?

Reply:
`ConfigurationState` is used when parsing available configurations from `configMgr`. `OsgiConfiguration` describes one particular config for single `pid`